onnxruntime/core/providers/qnn/builder/opbuilder/base_op_builder.cc

```diff
@@ -134,11 +133,11 @@ Status BaseOpBuilder::ProcessOutputs(QnnModelWrapper& qnn_model_wrapper,
     const auto& output_name = outputs[output_i].node_arg.Name();
 
     Qnn_QuantizeParams_t quantize_param = QNN_QUANTIZE_PARAMS_INIT;
-    utils::InitializeQuantizeParam(quantize_param, is_quantized_model);
+    utils::InitializeQuantizeParam(quantize_param, is_quantized_node);
```

Comment:
Similarly, we may be able to get rid of the `is_quantized_node` parameter in `ProcessOutputs` by inspecting the individual output: `bool is_quantized_output = output.quant_param.has_value()`

In fact, this is probably more accurate than just using `NodeUnit::Type::QDQGroup == node_unit.UnitType();`. I think we override ProcessOUtputs in some op builders just because only the input side is quantized, but not the output.